### ISSUE TITLE
Some of the game's menus do not respect (title?) safe area

### ISSUE THREAD

**Issue by alex-theman** [huguesjohnson/RetailClerk89#3]
If you're curious, the safe area is the region of the screen that most TV's, even really old and cheap ones, display. (in contrast to overscan, which most TVs don't display, at least to some extent) Some of the menus (like the in-game menu) and more obviously the brief "explainer" text for the controls, go outside this region. It looks something like this: (poor camera screenshot incoming)
   
![IMG_20200425_091841106 1](https://user-images.githubusercontent.com/1644295/80281285-07320d00-86d8-11ea-9db8-0c9a6a3ec33b.jpg)

While it doesn't really matter, (unless you are trying to play the game on a system with a display smaller than the Sega Genesis) most games tried to keep important details like dialogue or copyright notices inside this area back in the day. since they didn't know if they were being played on a "crappy" TV.  Since most of the ingame interface (including dialogue) stays inside this zone, (due to being copied from Phantasy Star 3) reformatting the text in some of the menus shouldn't be a big deal. If you want more specific details, [here's a nice explainer](http://wiki.nesdev.com/w/index.php/Overscan) from a NES homebrew wiki.

**Comment by huguesjohnson** [comment]
Thanks for noting this and sharing the link. True, I only tested this on emulators or Genesis hardware with a component cable so I never noticed this. I understand some prefer the original experience of playing on CRTs. However I don't think I'll fix this unless I decide to do a general overhaul of the game's graphics as this would also be an issue for all of the scenes which have borders along the edge.

**Comment by alex-theman** [comment]
Yeah, it's not a big deal, unless you want to go through a painful nostalgia trip, most people won't care as most modern TVs (including most late, standard def CRTs) behave fine. Speaking of painful nostalgia trips, what do the words Gravis Ultrasound bring to mind in your head?
